packages/svelte/src/constants.js

```diff
@@ -273,3 +273,30 @@ export function is_tag_valid_with_parent(tag, parent_tag) {
 
 	return true;
 }
+
+const ATTR_REGEX = /["<]/g;
+const CONTENT_REGEX = /[&<]/g;
+
+/**
+ * @template V
+ * @param {V} value
+ * @param {boolean} [is_attr]
+ */
+export function escape_html(value, is_attr) {
```

Comment:
this feels like an odd place for this function to live?

Reply:
I moved it to its own module